docs/checkers/index.md

```diff
@@ -2632,6 +2632,42 @@ Corrected version:
 print("Hello World!")  # This file ends with a single newline character! :)
 ```
 
+(W2301)=
+
+### Unnecessary ellipsis (W2301)
+
+This error occurs when a docstring is the preceding line of an ellipsis or if there is a statement
+in the same scope as an ellipsis
+
+```{literalinclude} /../examples/pylint/w2301_unnecessary_ellipsis.py
+
+```
+
+Corrected version:
+
+```python
+def my_func() -> None:
+    """Test Doctest"""
+    if True:
+        ...
+```
+
+(C2503)=
+
+### Bad file encoding (C2503)
+
+This error occurs when there is an encoding declaration or if any identifier is non-ASCII
```

Comment:
Change "encoding declaration" -> "encoding declaration at the top of the Python file".

Change "is non-ASCII" -> "uses non-ASCII characters".